### PR TITLE
Added fir.lic

### PR DESCRIPTION
**Pull request by skyleronken**
Added script to do the WM Fir Familiar quest

### CODE REVIEW

File: fir.lic [hunk 1]
```diff
@@ -0,0 +1,171 @@
+custom_require.call(%w[common common-crafting common-items common-money common-travel drinfomon equipmanager])
```
Comment (rpherbig) [inline fir.lic]: Would you mind adding a documentation block as a header?

File: fir.lic [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    bput('get water from vial','You get some holy water')
+    bput('stow vial','You put')
+    bput('put right hand in third hole','A foreboding wall moves out of your way to allow passage!')
+    bput('get my silver vial','You get')
+    bput('put water in my vial','You put')
+    bput('stow vial','You put')
+  end
+
+  def goto_old_man
+    return false if ['A voice whispers in your head',"You can't"].include? bput('west','A voice whispers in your head',"You can't",'Obvious exits')
```
Comment (rpherbig) [inline fir.lic]: I prefer the `when`/`case` style for these, but it's not a big deal.

File: fir.lic [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    when /fine details/
+      return
+    else
+      carve
+    end
+  end
+
+  def get_talisman(number)
+    number.times do |x|
+      return if 'I could not find' == bput('get fir bark', 'You get', 'I could not find')
```
Comment (rpherbig) [inline fir.lic]: If this fails, should `carve_talisman` also have a short circuit case?

File: fir.lic [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+      bput('give man', 'The old man takes your bark')
+      bput("stow right", 'You put')
+    end
+  end
+
+  def check_carving_knife
+    return if 'You tap' == bput('tap my carving knife','You tap','I could not find')
```
Comment (rpherbig) [inline fir.lic]: `common-items` has an `exists?` helper that might make this read a bit better

File: fir.lic [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    echo '**** You need a carving knife! ****'
+    beep
+    exit
+  end
+  def get_holy_water
+
+    return if 'You tap' == bput('tap my silver vial','You tap','I could not find')
```
Comment (rpherbig) [inline fir.lic]: Same thought on `exists?`

File: fir.lic [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+    bput('exp thievery', 'EXP HELP') if DRSkill.getrank('Thievery').zero?
```
Comment (rpherbig) [inline fir.lic]: This shouldn't be necessary since the requirement is so high - that `bput` is only needed for brand-new Thieves.

File: fir.lic [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+      buy_item(19073, "silver vial")
+    end
+
+    walk_to 5779
+    bput('fill my vial with water from basin','You fill your silver vial with some water.')
+    bput("stow my silver vial", 'You put')
+  end
+
+  def get_fir_bark(number)
+    return if 'You tap' == bput('tap my fir bark','You tap','I could not find')
```
Comment (rpherbig) [inline fir.lic]: Same thought on `exists?`

**Comment by rpherbig** [comment]
Would you be up for running `rubocop -a` on this file?

File: fir.lic (hunk 1 above)
Comment (skyleronken) [reply fir.lic]: done!

File: fir.lic (hunk 4 above)
Comment (skyleronken) [reply fir.lic]: done!

File: fir.lic (hunk 5 above)
Comment (skyleronken) [reply fir.lic]: done!

File: fir.lic (hunk 6 above)
Comment (skyleronken) [reply fir.lic]: removed!

File: fir.lic (hunk 7 above)
Comment (skyleronken) [reply fir.lic]: done!

File: fir.lic (hunk 3 above)
Comment (skyleronken) [reply fir.lic]: done!

**Comment by skyleronken** [comment]
@rpherbig Ran rubocop

**Pull request merged**